specs/MultiProfile.md

```diff
@@ -29,6 +29,18 @@ Users can use this interface to get the cookie manager, which is shared by all W
 with this profile. The cookie manager got from profile (CoreWebView2.Profile.CookieManager) is the
 same as that got from CoreWebView2 (CoreWebView2.CookieManager).
 
+Currently, we already have **ICoreWebView2Settings** interface to manage password-autosave and
+general-autofill, but it will work not immediately but after the next navigation, and it can only
+apply for current WebView2, which means if we start a new WebView2 using the same profile, all the
+settings are default values and cannot be set from the profile. By adding password-autosave and
+general-autofill management interfaces in profile, we can manage the properties and they will apply
+immediately if we set a new value, and all WebView2s that created with the same profile can share
+the settings, which means if we change password-autosave or general-autofill property in one WebView2,
+the others with the same profile will also work. And these two property is linked with the same
+properties in ICoreWebView2Settings, so changing one will change the other. So for the WebView2s
+with the same profile, their **IsPasswordAutosaveEnabled** or **IsGeneralAutofillEnabled** property
+in **CoreWebView2Settings** and **CoreWebView2Profile** should always keep in sync.
```

Comment:
This seems in conflict. You say the properties on the settings and the profile should always keep in-sync, but you also say the settings properties take effect on the next navigation and the profile settings take effect immediately. If the properties are applied at different times then it doesn't seem like they will always be in-sync. Or if they are always in-sync it doesn't seem like they can be applied at different times.

If you change the settings property, does the profile property only change after the settings property is applied during the next navigation? Or does the profile property change immediately when the settings property is changed? 

If they aren't always in-sync, which one 'wins' if both are set to different values?

Reply:
Good catch! Now the these two properties will work immediately no matter setting them in **CoreWebView2Settings** or **CoreWebView2Profile**, and when the property is changed in one interface, the same property in the other interface is changed as well immediately.

Reply:
This is potentially a compat breaking behavior change: Modifying the existing properties to set immediately instead of next navigation. Although for these properties in particular I would guess its unlikely to practically be a big problem. Also, it looks like this is already checked in like this for a while so that's fine. We'll just need to also update the existing docs for those properties to say that they take effect immediately (unlike most other things in Settings)

Reply:
Make sense, will update the existing docs, thanks!

Reply:
Yes, that's it. Please complete this PR, open a new PR to main, let me know, and I will schedule a review. Thanks!